### PR TITLE
Add updated README and docs

### PR DESCRIPTION
**Pull request by eemcmullan**
#277  
#274

### CODE REVIEW

File: READMEv2.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<h1 align="center"> OADP Operator v2 </h1>
```
Comment (kaovilai) [inline READMEv2.md:1]: I think we are calling the next version 0.3.0

File: controllers/vsl.go [hunk 2]
```diff
@@ -1,8 +1,9 @@
 package controllers
```
Comment (kaovilai) [inline controllers/vsl.go:1]: .go files shouldn't be part of this PR?

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: READMEv2.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+<p align="center">  OADP is OpenShift API for Data Protection operator. This operator sets up and 
+installs <a href="https://velero.io/">Velero</a> on the OpenShift platform. </p>
+
+<p align="center"><img src="https://img.shields.io/badge/Verison-2.0-yellow"/>
+<img src="https://img.shields.io/github/go-mod/go-version/openshift/oadp-operator"/>
+<img src="https://img.shields.io/badge/license-Apache_0.2-red" alt="license" />
```
Comment (kaovilai) [inline READMEv2.md:8]: Our license is 2.0

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: READMEv2.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    2. [Cloning the Repository](#clone-repo)
+    3. [Installing the Operator](#operator-install)
+    4. [Installing Velero + Restic](#velero-restic-install)
+    5. [Verify Installation](#verify-install)
+    6. [Uninstall Operator](#uninstall)
+2. [Custom Installation]()
+    1. [Configure Plugins](#docs/plugins.md)
+    2. [Backup Storage Locations and Volume Snapshot Locations](docs/bsl_and_vsl.md)
+    3. [Resource Requests and Limits](docs/resource_req_limits.md)
+    4. [Self-Sigend Certificate](docs/self_signed_certs.md)
```
Comment (apinnick) [inline READMEv2.md:24]: Sigend > Signed

File: controllers/vsl.go (hunk 2 above)
Comment (eemcmullan) [reply controllers/vsl.go:1]: @kaovilai my mistake. I was working on two things and forgot to switch branches. Good catch

File: READMEv2.md (hunk 3 above)
Comment (eemcmullan) [reply READMEv2.md:8]: @kaovilai I'll just remove all badges, since yours are nicer
